src/cache.h

```diff
@@ -5,10 +5,17 @@
 #include <utils/memutils.h>
 #include <utils/hsearch.h>
 
+#undef FLEXIBLE_ARRAY_MEMBER
```

Comment:
Why do you need to redefine this. Shouldn't pg_config.h set this correctly for the platform.

Reply:
It is not correctly set on the platform (it's not really platform-dependent, it's compiler dependent). On both OSX & Windows at least it is defined as empty.

Reply:
if you look at ./configure in postgres src it has logic for this. Why are we not hitting it?

Reply:
We don't use ./configure on Windows, and when they built the source they may have used a compiler than supports it?

Reply:
I would like @olofr and @erimatnor to look at this.

Reply:
I am not sure this is OK to do. Shouldn't this be set somewhere like `﻿﻿﻿src/tools/msvc/config_default.pl` as that seems to take `./configures` role under windows ?